README.md

```diff
@@ -1,122 +1,241 @@
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ focal main
+```
+Then run the following commands.
+
+```bash
+sudo apt update && apt-get install -y gnupg2
+wget -qO- https://packages.nlnetlabs.nl/aptkey.asc | sudo apt-key add -
+sudo apt update
+```
+
+You can then install, enable and start Krill by running
+
+```bash
+sudo apt install krill-sync
+```
+
+
+### Build with Cargo
 
 For Ubuntu 20.04 with Rust 1.51.0:
 
 ```
 $ apt update && apt install -y build-essential curl libssl-dev openssl pkg-config
 $ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 $ source $HOME/.cargo/env
-$ cargo install --git https://github.com/NLnetLabs/krill-sync.git --tag v0.1.2 --locked
-$ krill-sync https://<hidden fqdn of RRDP publication point>/rrdp/notification.xml -v
-...
+$ cargo install --git https://github.com/NLnetLabs/krill-sync.git --tag v0.2.0 --locked
 ```
 
-**Tada!** Rsync and RRDP directories have now been created locally which can be used to serve the content produced by the remote "hidden" RRDP origin server via the RRDP and Rsync protocols. Note: this tool does not serve them for you, for that you need NGINX and Rsyncd for example.
-
 ## Introduction
 
-This tool is intended to enable public facing RRDP and Rsync servers to serve consistent and fresh content obtained from a "hidden" RRDP master server, e.g. that runs a Krill RPKI Certificate Authority and RPKI publication repository. In this context "hidden" means that the server is not reachable by the public name of the repository but creates content that points to the public name. Krill Sync is smart enough to know that it shouldn't blindly follow FQDNs in the RRDP content to download snapshots and deltas but instead goes to the specified "hidden" FQDN each time instead.
+This tool is intended to enable public facing RRDP and Rsync servers to serve consistent
+and fresh content obtained from a "hidden" RRDP master server, e.g. that runs a Krill 
+RPKI Publication Server. In this context "hidden" means that the server is not reachable
+by the public name of the repository but creates content that points to the public name.
 
 ![Sequence Diagram](http://www.plantuml.com/plantuml/png/lPFVJzim4CVV_LUS-aGbYBGVJsYTg6WH4LE9qg9fKrzCVANMd7DcNu3yztCSGiOYDFgmlJJk-NFVEtzzFlCi7TUb4LNuwa9NaXWcEPf6qnra8TXCG7h8ivV4njMUOKx5the8RERYrZx-ALDPhzcw1jRexj4T-McdjiokNw9NJXi3wuQA25ojlwtE1PA2aUYbLWe9Hjenxp9TB9Oh8MOpI5Vf9fr_IR_FD-dr2cOp8Q8gd7n-mauXpnSPa6MzquLaRGbssY0u_5cZvoV-qybV-44tBWA63qupWYTP-RrbaAH0eM49DtdgN-aZ0YyQy4C7hv23LlJq5Bx6mhaFHWQLoW9RKDh_U8KBQgCkU-atf8wjwtFoh4tOp4zXeNrXm_r0dbsbu8Phy_7QmDWktuUSSDAG4nP3jHlV7yUwhYXRs24N-otz9bPOOYJkHUA09KaI9o5Rvkdr8ZqFqOjykRuyQ1Q2iDBjNsaEdGiuqZiezfBkK4s5a8OX10WVncXhXWuLg2vlqO7s8E-bDlBMeFXtumKxws8_SS-39TCSC79zF_EgCaslmHRkaD19HgWh5JxgkT4j7Tcgy_8O2acWoJDArTrdxAH7fwCg5_fnEF5U-dqmSmWyIFzSLxtrKS14iYrr8Orf9iUW4qosz2w1pbuE79RLbjud8OjaYL3KBgoutR96RUb3lnF94Ew9BNrCMaUGiIMd__7vZGuc198IP-6dBiql)
```

Comment:
This diagram no longer matches krill-sync behaviour.

Reply:
renoved it